docs/components/datastore.rst

```diff
@@ -1,169 +1,79 @@
-
-Instructions
-^^^^^^^^^^^^^
-1. Install and enabling the **geocoder** module.
-2. Click the **Manage Datastore** tab on any resource with address information.
-3. Check the "Geolocate" box.
-4. Select the Geolocation Service you will be using.
-5. In the Geolocate Addressses field enter the field or fields from the file that make up the address to geolocate.
-6. Click the **Import** button
-
-.. image:: ../images/datastore-geolocate.png
-
-Geolocation Services
-^^^^^^^^^^^^^^^^^^^^^
-Geolocation services offered are
-
-* `Google <https://developers.google.com/maps/articles/geocodestrat>`_
-* `Yahoo <http://developer.yahoo.com/boss/geo/>`_
-* `Nominatim <href="https://developer.mapquest.com/documentation/open/geocoding-api/>`_
-* `Yandex <http://api.yandex.com/maps/doc/geocoder/desc/concepts/input_params.xml>`_
-
-Note that Nominatim is a driven by `Open Street Map <http://www.openstreetmap.org/>`_ data, which is the most open of the options offered.
-
-Geolocation Limits
-^^^^^^^^^^^^^^^^^^^
-The number of rows that can be geolocated is determined by the service you select. Google, for example, allows you to geolocate up to 2500 times per day before paying.
+Importing Resources
+---------------
 
-Adding Service API Keys
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-The `Geocoder <https://drupal.org/project/geocoder>`_ module supports adding API keys for the Yahoo and Google services. Users can sign up for those services and, in Google's case, geocode up to 100,000 addressees per day.
+When viewing a resource you will see the "Manage Datastore" button at the top of the screen.
 
+Click the "Manage Datastore" button.
 
-Managing datastores with Drush
--------------------------------
-**To create a datastore from a local file:**
+Configure the Manager
+^^^^^^^^^^^^^^^
 
-.. code-block:: php
+There are many ways to accomplish data querability. DKAN out of the box provides two ways to manage the process. The default manager is called **Simple Import**.
 
-  drush dsc (path-to-local-file)
+The first step on using the datastore is to configure which manager you would like to use for the resource.
 
+On the "Manage Datastore" page, choose a datastore manager, and click save.
 
-**To update a datastore from a local file:**
+The **Simple Import** Manager will then need some information regarding the CSV format for proper parsing.
 
-.. code-block:: php
+- **delimiter**: the character that encloses the fields in your CSV file.
+- **escape**: the character used to escape other characters in your CSV file.
 
-  drush dsu (datastore-id) (path-to-local-file)
+Enter the character that are used as delimeter, quote and escape characters in your file, then use the "Import" button at the bottom of the page to import the data from your file or API into the datastore.
```

Comment:
Enter the character used for each function (delimeter, quote and escape) in your file, then click the "Import" button...